from revu.application.base.exception import CoreException


class ReviewModeException(CoreException):
    pass
